### ISSUE TITLE
Submission: psycho

### ISSUE THREAD

**Issue by DominiqueMakowski** [ropensci/onboarding#154]
### Summary

-   What does this package do? (explain in 50 words or less):

The goal of `psycho` is twofold: 1) provide functions for statistical routines crafted for the needs of psychological science and 2) provide tools to transform complex outputs (such as those of Bayesian and frequentist mixed models) into something that the user can (almost) directly paste into a report or a manuscript, promoting data exploration, analyses reproducibility and enhance the standardization of results reporting.

-   Paste the full DESCRIPTION file inside a code block below:

```
Package: psycho
Type: Package
Title: Efficient and Publishing-Oriented Workflow for Psychological Science
Version: 0.0.2
Authors@R: c(
    person("Dominique", 
        "Makowski", 
        email = "dom.makowski@gmail.com",
        role = c("aut", "cre")), 
    person("Viliam",
        "Simko",
        role = "ctb",
        email = "viliam.simko@gmail.com")
    )
Maintainer: Dominique Makowski <dom.makowski@gmail.com>
URL: https://github.com/neuropsychology/psycho.R
BugReports: https://github.com/neuropsychology/psycho.R/issues
Description: Toolbox for psychologists, neuropsychologists and neuroscientists. 
    It implements methods to create report-ready outputs for statistical models, 
    as well as various convenient statistical functions used in psychological science, 
    such as correlation matrices and normalization.
License: MIT + file LICENSE
Encoding: UTF-8
LazyData: true
RoxygenNote: 6.0.1
Depends: 
    R (>= 3.1.0)
Imports:
    ggplot2,
    dplyr,
    tidyr,
    purrr,
    tidyverse,
    rtf,
    psych,
    ppcor,
    corrplot,
    rstanarm,
    MuMIn,
    lmerTest
Suggests:
    knitr,
    rmarkdown,
    testthat,
    covr,
    lme4,
    broom
VignetteBuilder: knitr
```

-   URL for the package (the development repository, not a stylized html page):

https://github.com/neuropsychology/psycho.R

- Please indicate which category or categories from our [package fit policies](https://github.com/ropensci/onboarding/blob/master/policies.md#package-fit) this package falls under ***and why**(? (e.g., data retrieval, reproducibility. If you are unsure, we suggest you make a pre-submission inquiry.):

Data extraction and data munging as `psycho` extracts important indices, computes others, and cleans the output to present the data in the format needed by psychologists.

-   Who is the target audience?  

Neuroscientists and psychologists.

-   Are there other R packages that accomplish the same thing? If so, how does
yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?

There are no other R packages to my knowledge providing that set of tools.

### Requirements

Confirm each of the following by checking the box.  This package:

- [x] does not violate the Terms of Service of any service it interacts with. 
- [x] has a CRAN and OSI accepted license.
- [x] contains a README with instructions for installing the development version. 
- [x] includes documentation with examples for all functions.
- [x] contains a vignette with examples of its essential functions and uses.
- [x] has a test suite.
- [x] has continuous integration, including reporting of test coverage, using services such as Travis CI, Coeveralls and/or CodeCov.
- [x] I agree to abide by [ROpenSci's Code of Conduct](https://github.com/ropensci/onboarding/blob/master/policies.md#code-of-conduct) during the review process and in maintaining my package should it be accepted.

#### Publication options

- [x] Do you intend for this package to go on CRAN?  
- [x] Do you wish to automatically submit to the [Journal of Open Source Software](http://joss.theoj.org/)? If so:
    - [x] The package has an **obvious research application** according to [JOSS's definition](http://joss.theoj.org/about#submission_requirements).
    - [x] The package contains a `paper.md` matching [JOSS's requirements](http://joss.theoj.org/about#paper_structure) with a high-level description in the package root or in `inst/`.
    - [ ] The package is deposited in a long-term repository with the DOI: **I understood that this was done along with the publication, but should I do it before?**
    - (*Do not submit your package separately to JOSS*)

### Detail

- [x] Does `R CMD check` (or `devtools::check()`) succeed?  Paste and describe any errors or warnings:

- [x] Does the package conform to [rOpenSci packaging guidelines](https://github.com/ropensci/onboarding/blob/master/packaging_guide.md)? Please describe any exceptions:

- If this is a resubmission following rejection, please explain the change in circumstances:

- If possible, please provide recommendations of reviewers - those with experience with similar packages and/or likely users of your package - and their GitHub user names:

Unfortunately, I have no recommendation of reviewers.

**Comment by sckott** [comment]
hi @DominiqueMakowski 👋 

Thanks so so much for your submission!

Editors have discussed your submission and we regret to say that we've decided that it's out of scope. It seems to be a general workflow package, which doesn't fall under our criteria, because it's too general and doesn't hit the specific areas of reproducibility that we have decided to support. In addition, it implements statistical method which we don't support here. 

Let me know if you disagree or if you have any questions.
